phenopy/score.py

```diff
@@ -40,7 +40,15 @@ def find_lca(self, term_a, term_b):
         common_parents = parents[0].intersection(
             parents[1])
         # lca node
-        return max(common_parents, key=lambda n: self.hpo_network.node[n]['ic'])
```

Comment:
@vgainullin do you think this is worth it, or should we just revert to line 43 just grabbing the greatest `depth` ancestor